store/tikv/store_test.go

```diff
@@ -338,11 +338,16 @@ type checkRequestClient struct {
 }
 
 func (c *checkRequestClient) SendReq(ctx goctx.Context, addr string, req *tikvrpc.Request) (*tikvrpc.Response, error) {
+
+	resp, err := c.Client.SendReq(ctx, addr, req)
 	if c.priority != req.Priority {
-		return nil, errors.New("request check error")
+		if resp.Get != nil {
```

Comment:
`return error` directly, is different from returning a response and the response has an error field.
I want simulate the later, so I set the response field.

Response struct is a bit complex, it may be Get/Prewrite/Commit/Scan/Rollback and so on, and it's Get here.